docs/installation.md

```diff
@@ -119,12 +126,26 @@ where GCP_PAS_NETWORK is the name of the network used when creating the private
 
 ### Fetch A Broker and GCP Brokerpak
 
-Download a release from https://github.com/pivotal/cloud-service-broker/releases. Find the latest release matching the name pattern `sb-0.1.0-rc.XXX-gcp-0.0.1-rc.YY`. This will have a broker and brokerpak that have been tested together. Follow the hyperlink into that release and download `cloud-servic-broker` and `gcp-services-0.1.0-rc.YY.brokerpak` into the same directory on your workstation.
+Download a Cloud Service Broker release from https://github.com/pivotal/cloud-service-broker/releases. 
```

Comment:
should be `cloudfoundry` instead of `pivotal`. It redirects but it is just not the right url